src/build/mod.rs

```diff
@@ -165,6 +168,11 @@ async fn process_dockerfile<R: AsyncRead + std::marker::Unpin>(
             last_entrypoint = Some(directive.clone());
         } else if let Directive::Expose { port } = directive {
             exposed_port = *port;
+        } else if let Directive::User(b) = directive{
+            if let Ok(user) = String::from_utf8(b.to_vec()) {
```

Comment:
should we maybe return a build error here if we can't parse the user directive?

Reply:
yeah throwing an error here makes sense